ui/components/component-library/modal/modal.stories.tsx

```diff
@@ -1,21 +1,21 @@
 import React, { useState } from 'react';
```

Comment:
Updating `Modal` stories to use the `ModalBody` component. Will be making some further updates to the Modal to handle scrolling inside of the modal rather than using the browsers scrolling behaviour